package.json

```diff
@@ -1,37 +1,40 @@
 {
   "name": "grunt-requiresafe",
-  "version": "1.1.2",
   "description": "Run requireSafe as a grunt task",
-  "main": "index.js",
-  "scripts": {
-    "test": "grunt",
-    "shrinkwrap": "npm shrinkwrap && shrinkydink"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+ssh://git@github.com:requiresafe/grunt-requiresafe.git"
-  },
-  "keywords": [
-    "grunt",
-    "security",
-    "requiresafe",
-    "nodesecurity",
-    "grunt",
-    "gruntplugin"
-  ],
+  "version": "1.1.2",
   "author": "^Lift Security",
-  "license": "Apache-2.0",
   "bugs": {
     "url": "https://github.com/requiresafe/grunt-requiresafe/issues"
   },
-  "homepage": "https://github.com/requiresafe/grunt-requiresafe#readme",
   "dependencies": {
     "chalk": "^1.1.1",
     "cli-table": "^0.3.1",
     "requiresafe": "^2.1.1"
   },
   "devDependencies": {
+    "eslint": "^1.7.2",
+    "eslint-config-requiresafe": "^1.0.0",
     "grunt": "^0.4.5",
     "shrinkydink": "^1.0.0"
+  },
+  "homepage": "https://github.com/requiresafe/grunt-requiresafe#readme",
+  "keywords": [
+    "grunt",
+    "security",
+    "requiresafe",
+    "nodesecurity",
+    "grunt",
+    "gruntplugin"
+  ],
+  "license": "Apache-2.0",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com:requiresafe/grunt-requiresafe.git"
+  },
+  "scripts": {
+    "lint": "eslint .",
```

Comment:
Added this script since it's tricky to call eslint otherwise due to the global ESLint and local configs.

Sorry about the rest of the noise in package.json, I ran the **fixpack** module to sort things.